.github/CODEOWNERS

```diff
@@ -1 +1 @@
-* @uniswap/protocols @jsy1218 @dannydaniil @cgkol
```

Comment:
can we also keep @cgkol here? I understand team alias would be nice, but I think not entire @Uniswap/routing team is up to sdks codeowner yet.

Reply:
@cgkol is in APIs team